grammar.js

```diff
@@ -124,7 +125,7 @@ module.exports = grammar({
     predicate: $ =>
       seq(
         "(",
-        field("name", seq("#", $._immediate_identifier, field("type", $.predicate_type))),
+        field("name", seq(choice("#", "."), $._immediate_identifier, field("type", $.predicate_type))),
```

Comment:
If we don't need to capture it
```suggestion
        field("name", seq(/[#.]/, $._immediate_identifier, field("type", $.predicate_type))),
```

Reply:
Sadly still do, for some reason scanner accepts it if not first character, eg `.ma.e-range` is valid

Reply:
Sorry I just woke up and did not process your review :) I misunderstood. 

But I do think it would be nice to capture it still so it can be highlighted